include/eve/module/core/regular/impl/simd/arm/sve/logical_and.hpp

```diff
@@ -0,0 +1,22 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/category.hpp>
+#include <eve/detail/implementation.hpp>
+
+namespace eve::detail
+{
+  template<callable_options O, typename T, typename U, typename N>
+  EVE_FORCEINLINE logical<wide<T, N>> logical_and_(EVE_REQUIRES(sve_), O const&, logical<wide<T, N>> v, logical<wide<U, N>> w) noexcept
+    requires(sve_abi<abi_t<T, N>> || sve_abi<abi_t<U, N>>)
+  {
+    return svmov_z(v, convert(w, as<logical<T>>{}));
```

Comment:
this doesn't seem right anymore. I appreciate that you might not had time to fix it and this is a draft.

Reply:
The return type shouldn't change based on what logical is first